### DISCUSSION TITLE
Server is in maintenance mode, please try again in a few minutes.

### DISCUSSION THREAD

**Discussion by janaxhell** [nilsteampassnet/TeamPass#4275] [Q&A]
I have Gokapi installed as a docker container. I don't use it very often, but I expect it to be available when I need it.
Instead every single time I open it, I get 
**Server is in maintenance mode, please try again in a few minutes.**
and I have to reconfigure it again and again.
I think this happens at the beginning of every month, but I'm not sure. 
Is there a way to completely avoid this and set it and forget it?

**Comment by ygypt** [comment]
Just spun up a docker for the first time, same issue
